client/src/components/teams/MembersToBeInvited.js

```diff
@@ -0,0 +1,67 @@
+import {
+  Typography,
+  Grid,
+  Button,
+  makeStyles,
+  InputLabel,
+} from "@material-ui/core";
+import { Brightness1 } from "@material-ui/icons";
+import React from "react";
+
+const MembersToBeInvited = ({ teamMembers, setTeamMembers }) => {
+  const classes = useStyles();
+  const teamMemberNotEmpty = teamMembers.length !== 0;
+  const removeEmailFromInvitedList = (memberToRemove) => () => {
+    // Caveat: each email must be unique in the list for this to work
+    const filteredList = teamMembers.filter(
+      (member) => member.email !== memberToRemove.email
+    );
+    setTeamMembers(filteredList);
+  };
+  return (
+    <Grid container direction="column" className={classes.root}>
+      <Grid item>
+        <InputLabel className={classes.label}>
+          An invitation email will be sent to the following members:
+        </InputLabel>
+      </Grid>
+      <Grid item container direction="column" wrap="nowrap">
+        {teamMemberNotEmpty &&
+          teamMembers.map((member, index) => (
+            <Grid item container key={index} alignItems="center" spacing={3}>
```

Comment:
key should be something unique like the email to be safer. Otherwise if there's another map using index, or the order of the members changes, etc. It could cause re-render issues

Reply:
This makes sense, I will use email instead.